// html-validate/dist/esm/index.js
export { F as FileSystemConfigLoader, a as FlatConfigLoader, H as HtmlValidate, c as cjsResolver, b as compatibilityCheck, d as defineFlatConfig, e as esmResolver } from './core-nodejs.js';
export { A as Attribute, C as Config, a as ConfigError, b as ConfigLoader, D as DOMNode, c as DOMTokenList, d as DOMTree, e as DynamicValue, f as EventHandler, H as HtmlElement, M as MetaCopyableProperty, g as MetaTable, N as NestedError, h as Node, P as Parser, i as PerformanceTracker, R as Reporter, j as ResolvedConfig, k as Rule, S as SchemaValidationError, l as Severity, m as StaticConfigLoader, T as TextClassification, n as TextContent, o as TextNode, U as UserError, W as WrappedError, p as ariaNaming, r as classifyNodeText, a2 as configPresets, G as defineConfig, I as definePlugin, O as formatterFactory, Y as isUserError, _ as keywordPatternMatcher, a3 as ruleExists, a4 as sliceLocation, a5 as staticResolver, aa as version, ac as walk } from './core.js';
export { C as CLI } from './cli.js';
export { d as defineMetadata, m as metadataHelper } from './meta-helper.js';
import 'node:fs';
import 'node:path';
import 'node:fs/promises';
import 'node:url';
import 'node:module';
import 'kleur';
import 'ajv';
import './elements.js';
import './utils/parse-image-candidate-string.js';
import '@sidvind/better-ajv-errors';
import './utils/natural-join.js';
import '@html-validate/stylish';
import 'semver';
import 'node:path/posix';
import 'prompts';
import 'node:util';
