// html-validate/dist/esm/index.js
export { F as FileSystemConfigLoader, H as HtmlValidate, c as cjsResolver, a as compatibilityCheck, e as esmResolver, n as nodejsResolver } from './core-nodejs.js';
export { A as Attribute, g as Config, f as ConfigError, C as ConfigLoader, D as DOMNode, o as DOMTokenList, p as DOMTree, q as DynamicValue, r as EventHandler, H as HtmlElement, M as MetaCopyableProperty, s as MetaTable, N as NestedError, u as NodeClosed, w as NodeType, a as Parser, P as PerformanceTracker, R as Reporter, x as ResolvedConfig, y as Rule, z as SchemaValidationError, B as Severity, S as StaticConfigLoader, T as TextClassification, F as TextContent, G as TextNode, U as UserError, V as Validator, W as WrappedError, I as ariaNaming, J as classifyNodeText, K as configPresets, L as defineConfig, O as definePlugin, k as formatterFactory, Q as isUserError, X as keywordPatternMatcher, Y as ruleExists, Z as sliceLocation, _ as staticResolver, v as version, $ as walk } from './core.js';
export { C as CLI } from './cli.js';
export { d as defineMetadata, m as metadataHelper } from './meta-helper.js';
import 'node:fs';
import 'node:path';
import 'node:fs/promises';
import 'node:url';
import 'node:module';
import 'kleur';
import 'ajv';
import './elements.js';
import './utils/parse-image-candidate-string.js';
import '@sidvind/better-ajv-errors';
import './utils/natural-join.js';
import '@html-validate/stylish';
import 'semver';
import 'node:path/posix';
import 'glob';
import 'prompts';
